setup.cfg

```diff
@@ -0,0 +1,4 @@
+[flake8]
+# 201 and 202 are spaces after ( and before ), 501 is line length. These
+# are exceptions allowed (encouraged?) by Galaxy style guidelines.
```

Comment:
https://wiki.galaxyproject.org/Develop/BestPractices
Adding to GH is a good idea. I will bounce it around.